### PR TITLE
Get rid of 'test' queues which weren't needed

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
We were modifying the Rabbit queue config from the 'production-like' config, therefore invalidating our testing.

# What has changed
Used the proper queues instead of the invented ones which should not and now do not exist anywhere.

# How to test?
Run the acceptance tests.

# Links
Trello: https://trello.com/c/NG6W8Vzy

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Strip out test queues from CI environment - not needed - and use real queues](https://trello.com/c/NG6W8Vzy/520-strip-out-test-queues-from-ci-environment-not-needed-and-use-real-queues)

**Review by ryangrundy7 — APPROVED** [review]

File: config.py [hunk 1]
```diff
@@ -67,12 +67,9 @@ class Config:
...
     # For test queues
-    RABBITMQ_RH_OUTBOUND_CASE_QUEUE_TEST = 'case.rh.case.test'
-    RABBITMQ_CASE_TEST_ROUTE = os.getenv('RH_CASE_ROUTING_KEY', 'event.case.*')
-    RABBITMQ_RH_OUTBOUND_UAC_QUEUE_TEST = 'case.rh.uac.test'
-    RABBITMQ_UAC_TEST_ROUTE = os.getenv('RH_UAC_ROUTING_KEY', "event.uac.*")
-    RABBITMQ_RH_EXCHANGE_NAME = os.getenv('RH_EXCHANGE_NAME', "events")
-    RABBITMQ_OUTBOUND_FIELD_QUEUE_TEST = 'RM.Field.Test'
+    RABBITMQ_RH_OUTBOUND_CASE_QUEUE_TEST = 'case.rh.case'
+    RABBITMQ_RH_OUTBOUND_UAC_QUEUE_TEST = 'case.rh.uac'
+    RABBITMQ_OUTBOUND_FIELD_QUEUE_TEST = 'RM.Field'
```
Comment (AdamHawtin) [inline config.py]: Should remove `_TEST` from these config names, that's how we were distinguishing test queues from normal ones.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Looks fine apart from I think the config vars should be renamed to keep up with the changes.

Also perhaps a line in the README somewhere saying the tests won't work in any environment that has consumers on those queues

File: config.py (hunk 1 above)
Comment (NickGrantX) [reply config.py]: ugh. I guess so <sigh>

**Comment by NickGrantX** [comment]
> Looks fine apart from I think the config vars should be renamed to keep up with the changes.
> 
> Also perhaps a line in the README somewhere saying the tests won't work in any environment that

The tests would never have worked anywhere except CI and local machines, because they purged normal queues anyway

**Review by AdamHawtin — APPROVED** [review]

**Comment by liamtoozer** [comment]
[RMC-317 Make Hand Delivery flag in outbound field messages dynamic based on treatment code (8)](https://trello.com/c/esdStYzJ/514-rmc-317-make-hand-delivery-flag-in-outbound-field-messages-dynamic-based-on-treatment-code-8)

**Pull request merged**